docs/book/src/concepts.md

```diff
@@ -0,0 +1,109 @@
+# Concepts
+
+## How it works
+
+The diagram below illustrates how Secrets Store CSI volume works:
+
+![diagram](../images/diagram.png)
+
```

Comment:
Below the diagram, can we also add an explanation to how it works? Something like

```suggestion
Similar to Kubernetes secrets, on pod start and restart, the Secrets Store CSI driver calls the provider binary to retrieve the secret content from the external Secrets Store specified in the `SecretProviderClass` custom resource. Then the volume is mounted in the pod as `tmpfs` and the secret contents are written to the volume.
```

Reply:
Q: should we not say provider `binary` anymore since grpc is going to be the standard going forward.

Reply:
Good point. I'll update it!

Reply:
Can you add the explanation I suggested below the diagram?

Reply:
I missed the explanation. Added it to the doc.